tests/test_streaming.py

```diff
@@ -175,6 +175,22 @@ def on_read(chunk_size):
         # The mocked function not have been called at all since the stream looks closed
         self.assertEqual(mock_read.call_count, 0)
 
+    def test_read_incomplete_buffer(self):
+        from six.moves import http_client as httplib
+        for length in [1, 2, 5, 10, 20, 50]:
+            with self.assertRaises(httplib.IncompleteRead):
+                msg = "11\n".reaplce('\n', '')
```

Comment:
You meant `replace`. Seems like the file isn't in `travis-tests.cfg` so this file isn't run.